centreon/tests/e2e/cypress/e2e/Dashboards/05-dashboard-check-permissions/index.ts

```diff
@@ -0,0 +1,783 @@
+import { Given, When, Then } from '@badeball/cypress-cucumber-preprocessor';
+
+import dashboards from '../../../fixtures/dashboards/check-permissions/dashboards.json';
+import adminUser from '../../../fixtures/users/admin.json';
+import dashboardAdministratorUser from '../../../fixtures/users/user-dashboard-administrator.json';
+import dashboardCreatorUser from '../../../fixtures/users/user-dashboard-creator.json';
+import dashboardViewerUser from '../../../fixtures/users/user-dashboard-viewer.json';
+
+before(() => {
+  cy.startWebContainer();
+  cy.execInContainer({
+    command: `sed -i 's@"dashboard": 0@"dashboard": 3@' /usr/share/centreon/config/features.json`,
+    name: Cypress.env('dockerName')
+  });
+  cy.executeCommandsViaClapi(
+    'resources/clapi/config-ACL/dashboard-check-permissions.json'
+  );
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/dashboards?'
+  }).as('listAllDashboards');
+  cy.intercept({
+    method: 'POST',
+    url: '/centreon/api/latest/configuration/dashboards'
+  }).as('createDashboard');
+  cy.loginByTypeOfUser({
+    jsonName: adminUser.login,
+    loginViaApi: true
+  });
+  cy.insertDashboard({ ...dashboards.fromAdminUser });
```

Comment:
```suggestion
  cy.insertDashboard(dashboards.fromAdminUser);
```
This should work too